systemverilog-plugin/UhdmAst.cc

```diff
@@ -3615,6 +3615,20 @@ void UhdmAst::process_func_call()
     });
 }
 
+void UhdmAst::process_task_call()
+{
+    current_node = make_ast_node(AST::AST_TCALL);
```

Comment:
It seems that apart from this type, the processing is identical to function calls (and it also handles `localparam`s). If that's true, it would make sense to handle it in a common function, like `process_tf_call`.

Reply:
Thank you for your feedback! Alright, I've merged the two functions into `process_tf_call`.